pkg/util/object-identifier.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2023 The KubeStellar Authors.
```

Comment:
FYI, it's a new year.

Reply:
We do not have to update all our files every year. OTOH, it is customary to write the current year in a new file.

Reply:
Understandable, done.